### ISSUE TITLE
Lag spikes when moving around the map

### ISSUE THREAD

**Issue by FlamingCheese** [tgstation/tgstation#74200]
## Reproduction:
To reproduce this you simply have to connect to a server that is far away for you and move around the station or you can watch the video below
https://streamable.com/do0wuu - the game spikes up from an average of 220ms to over 1000 when moving around the station and switching z-levels takes a lot of time (I have set my multiz quality setting to the lowest after recording with no effectiveness of removing the problem), also this happens on non multi-z maps as well with no noticable change in effect. This also occurs if I am connected to terry (around 50ms for me) with somewhat less noticeable but still occurring lag spikes 
I have not seen a similar issue report on this but I have seen others comment that they experience the same as well. My best bet on why this happens is something to do with planecube (notably after #69115) but that is mainly just a guess so I could be wrong. As another note, this doesn't occur on other servers that run /tg/ based code (the ones that don't have the multiz rework pr merged, or at least that is what a cursory search showed)
~Better titles are appreciated for suggestion by the way~

**Comment by FlamingCheese** [comment]
I have a 3070Ti which should be way more than capable of running this cursed game unless its something to do with the 30 series, but I do not have another pc to test this out on to see if its the case

**Comment by TheVekter** [comment]
I'm gonna close this out as a duplicate of #72206.
